uvm/impl/com/untangle/uvm/UriManagerImpl.java

```diff
@@ -326,6 +326,10 @@ private UriManagerSettings defaultSettings()
         uriTranslation.setUri("https://launchpad.edge.arista.com/");
         uriTranslations.add(uriTranslation);
 
+        uriTranslation = new UriTranslation();
+        uriTranslation.setUri("https://wiki.edge.arista.com/get.php");
+        uriTranslations.add(uriTranslation);
+
         settings.setUriTranslations(uriTranslations);
```

Comment:
Looks good!   

But you also need to add in the updateSettings function for upgrades and increase the SettingsCurrentVersion.

Reply:
@cblaise-untangle added suggested changes.